faster_whisper/utils.py

```diff
@@ -77,13 +77,20 @@ def download_model(
         "tokenizer.json",
         "vocabulary.txt",
     ]
+    kwargs["allow_patterns"] = allow_patterns
+    kwargs["tqdm_class"] = disabled_tqdm
 
-    return huggingface_hub.snapshot_download(
-        repo_id,
-        allow_patterns=allow_patterns,
-        tqdm_class=disabled_tqdm,
-        **kwargs,
-    )
+    try:
+        return huggingface_hub.snapshot_download(
+            repo_id,
+            **kwargs,
+        )
+    except huggingface_hub.utils._errors.HfHubHTTPError:
```

Comment:
You can use the public symbol here.

```suggestion
    except huggingface_hub.utils.HfHubHTTPError:
```

https://huggingface.co/docs/huggingface_hub/v0.14.1/en/package_reference/utilities#huggingface_hub.utils.HfHubHTTPError